public/fixtures/image-text/slide5.json

```diff
@@ -1,14 +1,20 @@
 {
-  "id": "textBox5",
+  "id": "imageText5",
   "title": "Slide 5",
   "template": "template-image-text",
   "duration": 15000,
   "content": {
     "title": "Slide 5",
     "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.",
-    "boxAlign": "bottom",
     "backgroundColor": "rgb(255, 0, 0)",
     "textColor": "rgb(0, 0, 255)",
-    "boxColor": "rgb(0, 255, 0)"
+    "boxColor": "rgb(0, 255, 0)",
```

Comment:
What is the relation here between "styling" element and the style stuff outside it ?

Reply:
Its old, and i figured it would be removed or changed when the theme was implemented.